cdc/tests/integration_tests/multi_capture/run.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -eu
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=tikv-cdc.test
+SINK_TYPE=$1
+UP_PD=http://$UP_PD_HOST_1:$UP_PD_PORT_1
+DOWN_PD=http://$DOWN_PD_HOST:$DOWN_PD_PORT
+
+Start_Key0=indexInfo_:_pf01_:_APD0101_:_0000000000000000000
```

Comment:
How about just define 2 split keys ?
Then the 3 ranges would be [, SplitKey1), [SplitKey1, SplitKey2), [SplitKey2, )

Reply:
I removed all StartKey(n) and EndKey(n), added StartKey, SplitKey1, SplitKey2 and EndKey.